packages/bot/src/stores/quick-strategy-store.js

```diff
@@ -2,33 +2,41 @@ import {
     observable,
     action,
     runInAction,
-}                from 'mobx';
-import ApiHelper from '../services/api/api-helpers';
-import config    from '../constants/index';
-import { load }  from '../scratch/utils';
+}                    from 'mobx';
+import ApiHelper     from '../services/api/api-helpers';
+import config        from '../constants/index';
+import { load }      from '../scratch/utils';
+import { translate } from '../utils/lang/i18n';
 
 export default class QuickStrategyStore {
     constructor(root_store) {
         this.root_store = root_store;
     }
 
     initial_values = {
-        symbol   : 'WLDAUD',
-        tradetype: 'callput',
-        stake    : '',
-        size     : '',
-        loss     : '',
-        profit   : '',
+        symbol       : 'WLDAUD',
+        trade_type   : 'callput',
+        duration_type: 't',
+        duration     : 5,
+        stake        : '',
+        size         : '',
+        loss         : '',
+        profit       : '',
     };
 
     @observable is_strategy_modal_open = false;
     @observable active_index           = 0;
     @observable market_dropdown        = {};
     @observable trade_type_dropdown    = {};
+    @observable duration_dropdown      = [];
 
     @action.bound
     toggleStrategyModal() {
         this.is_strategy_modal_open = !this.is_strategy_modal_open;
+
+        if (this.is_strategy_modal_open) {
+            this.onMount();
```

Comment:
Better to rename this to something like `onOpenModal`, we use `onMount` for when a React component is mounted.